modules/tinymce/src/themes/silver/test/ts/browser/editor/color/ColorPickerSanityTest.ts

```diff
@@ -97,6 +97,12 @@ describe('browser.tinymce.themes.silver.editor.color.ColorPickerSanityTest', ()
 
       it('TBA: Open dialog, pick a different color, click Cancel and assert color does not change', async () => {
         const editor = hook.editor();
+        // Change color to black
+        await pOpenDialog(editor);
+        await pSetHexBlack(editor);
+        TinyUiActions.submitDialog(editor);
+        await pWaitForDialogClose(editor);
```

Comment:
Actually... thinking about this more, should just do a `setContent` call instead?

Reply:
The way the test file is set up, it looks like it doesn't use what is present in the editor at all. I'll confirm that though

Reply:
Looks like that is the case. I think I might just change the test message to more accurately point out what is happening. Thanks for pointing that out